File: sentiment-analysis/google-cloud-natural-language/GoogleCloudNaturalLanguageSentimentAnalysis.java
try (LanguageServiceClient language = LanguageServiceClient.create()) {
    Document document = Document.newBuilder()
        .setContent(text)
        .setType(Document.Type.PLAIN_TEXT)
        .build();

    Sentiment sentiment = language
        .analyzeSentiment(document)
        .getDocumentSentiment();
}

File: sentiment-analysis/google-cloud-natural-language/GoogleCloudNaturalLanguageSentimentAnalysis.kt
// "content" is a String containing the input document in the specified type.

val client = LanguageServiceClient.create()

val document = Document.newBuilder()
    .setContent(content)
    .setType(Document.Type.PLAIN_TEXT)
    .build()

val sentiment = client
    .analyzeSentiment(document)
    .documentSentiment
